man/systemctl.xml

```diff
@@ -876,6 +876,18 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
             a similar filter for messages and might be more
             convenient.
             </para>
+            
+            <para>The Loaded: line in the output will show "loaded" if the unit has been loaded into memory and "not-found" if service can't be found.
+              Along with showing the path to the unit file, this line will also show the enablement state. 
+              Enabled commands start at boot. 
+              See the full table of possible enablement stats in documentation for the is-enabled command. </para>
+            <para>The Active: line shows active state. 
+              The value is usually "active" or "inactive". Active could mean started, bound, plugged in, etc depending on the unit type.
```

Comment:
Yes, please try to avoid duplication. Things like that tend to get out of sync if described in more than one place. Referring the reader to the other section sounds like a better idea.